### PR TITLE
docs(subscriptions): updated subscriptions docs

### PR DESCRIPTION
**Pull request by dotansimha**

### CODE REVIEW

File: tools/graphql-subscriptions/index.md [hunk 1]
```diff
@@ -3,13 +3,14 @@ title: Installation
...
 
-`graphql-subscriptions` is a transport-agnostic JavaScript utility that helps you execute GraphQL subscription on your NodeJS server. `subscriptions-transport-ws` is a WebSocket server and client library for GraphQL subscriptions that complements `graphql-subscriptions` and can be used directly in a JavaScript app or wired up to a fully-featured GraphQL client like Apollo or Relay.
+We will use the `PubSub` implementation from `graphql-subscriptions`, and we will connect it to `subscribe` executor of `graphql`, and publish the data using `subscriptions-transport-ws` (a WebSocket server and client library for GraphQL that can be used directly in a JavaScript app or wired up to a fully-featured GraphQL client like Apollo or Relay.
 
 The process of setting up a GraphQL subscriptions server consist of the following steps:
 
 1. Declaring subscriptions in the GraphQL schema
 2. Setup a PubSub instance that our server will publish new events to
-3. Connecting the PubSub mechanism to the schema with `SubscriptionManager` that will receive publications from the PubSub and resolve them through GraphQL executer
+3. Hook together `PubSub` evnet and GraphQL subscription.
```
Comment (stubailo) [inline tools/graphql-subscriptions/index.md]: "evnet" -> "event"

File: tools/graphql-subscriptions/lifecycle-events.md [hunk 2]
```diff
@@ -7,8 +7,8 @@ order: 404
 
 * `onConnect` - called upon client connection, with the `connectionParams` passed to `SubscriptionsClient` - you can return a Promise and reject the connection by throwing an exception. The resolved return value will be appended to the GraphQL `context` of your subscriptions.
 * `onDisconnect` - called when the client disconnects.
-* `onSubscribe` - called when the client subscribes to GraphQL subscription - use this method to create custom params that will be used when resolving this subscription.
-* `onUnsubscribe` - called when client unsubscribes from a GraphQL subscription.
+* `onOperation` - called when the client executes a GraphQL operation - use this method to create custom params that will be used when resolving the operarion.
```
Comment (stubailo) [inline tools/graphql-subscriptions/lifecycle-events.md]: operarion -> operation

**Comment by dotansimha** [comment]
Thanks @stubailo ! fixed!

**Comment by Urigo** [comment]
thanks @dotansimha

**Pull request merged**